maquetado/scss/styles.scss

```
@@ -13,6 +13,16 @@
   height: 368px;
   width: 261px;
   margin-right: 40px;
+  position: relative;
+}
+
+.book-badge {
+  position: absolute;
+  height: 100px;
+  width: 80px;
+  left: 226px;
```

Same, I would set this as `::after` or `::before`. Also, remember to organize CSS properties in alphabetical order. Last but not least,  we could use right for this. This number is too big and it makes it hard to guess how it will look by just watching the property. 

If not, we could do: 

```suggestion
left: calc(100% - some low value); 
```